vidarr-cromwell/src/main/java/ca/on/oicr/gsi/vidarr/cromwell/CromwellOutputProvisioner.java

```diff
@@ -52,7 +52,9 @@
           new Pair<>(".seg", "application/seg"),
           new Pair<>(".Rdata", "application/rdata"),
           new Pair<>(".RData", "application/rdata"),
-          new Pair<>("", "application/octet-stream"));
+          new Pair<>("", "application/octet-stream"),
+          new Pair<>(".cram", "application/cram"),
```

Comment:
.cram and .crai need to be above the `"" --> "application/octet-stream"` entry as the EXTENSION_TO_META_TYPE list is used to find the first matching extension in the list.